dk.bruchhaus.FFQueue.yml

```diff
@@ -0,0 +1,69 @@
+id: dk.bruchhaus.FFQueue
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+runtime-version: '49'
+command: ffqueue
+rename-icon: ffqueue
+rename-desktop-file: ffqueue.desktop
+rename-appdata-file: ffqueue.xml
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos
+  - --filesystem=/media
+  - --filesystem=/mnt
+
+cleanup:
+  - /lib/pkgconfig
+  - /bin/wx*
+  - /include
+  - /lib/debug
+  - /lib/wx
+
+modules:
+
+  - name: FFmpeg
+    buildsystem: simple
+    build-commands:
+      - mkdir -p $FLATPAK_DEST/bin
+      - mkdir -p $FLATPAK_DEST/lib
+      - cp -Pr bin/* $FLATPAK_DEST/bin/
+      - cp -Pr lib/* $FLATPAK_DEST/lib/
+    sources:
+      - type: archive
+        url: https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2025-09-30-13-19/ffmpeg-N-121271-g74115b017c-linux64-gpl-shared.tar.xz
```

Comment:
This should be built from source.

https://docs.flathub.org/docs/for-app-authors/requirements#building-from-source

Also see the comment above about being in the runtime.

Reply:
this is not "other flatpak's" we are reviewing.